### PR TITLE
Add tracktime task

### PR DESCRIPTION
**Pull request by njacazio**

### CODE REVIEW

**Comment by njacazio** [comment]
This PR handles the changes reported in https://github.com/AliceO2Group/AliceO2/pull/7048

File: ALICE3/TableProducer/alice3-pidTOF.cxx [hunk 1]
```diff
@@ -90,7 +90,8 @@ struct ALICE3pidTOFTask {
   float nsigma(Trks::iterator track)
   {
     // return (track.tofSignal() - tof::ExpTimes< Trks::iterator, id>::GetExpectedSignal(track.collision(), track)) / sigma<id>(track);
-    return (track.tofSignal() - track.collision().collisionTime() * 1000.f - tof::ExpTimes<Trks::iterator, id>::GetExpectedSignal(track)) / sigma<id>(track);
+    // return (track.tofSignal() - track.collision().collisionTime() * 1000.f - tof::ExpTimes<Trks::iterator, id>::GetExpectedSignal(track)) / sigma<id>(track);
```
Comment (jgrosseo) [inline ALICE3/TableProducer/alice3-pidTOF.cxx:93]: remove the comments

File: Common/TableProducer/PID/pidTOFTrackTime.cxx [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
+{
+  std::vector<ConfigParamSpec> options{{"add-qa", VariantType::Int, 0, {"Produce TOF PID QA histograms"}}};
+  std::swap(workflowOptions, options);
+}
+
+#include "Framework/runDataProcessing.h"
+
+struct trackTime {
```
Comment (jgrosseo) [inline Common/TableProducer/PID/pidTOFTrackTime.cxx:45]: When this is moved to the pidFull task, this executable can be renamed to contain qa

**Comment by njacazio** [comment]
The issues in the compilation are due to the fact that the O2 has still the field for the `tofSignal`

**Comment by jgrosseo** [comment]
Replaced by https://github.com/AliceO2Group/O2Physics/pull/93

**Pull request closed without merging**